src/components/protocol_handler/src/protocol_packet.cc

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, Ford Motor Company
+ * Copyright (c) 2018, Ford Motor Company
```

Comment:
@ydementieiev 2019

Reply:
@AByzhynar  done in [5148131](https://github.com/smartdevicelink/sdl_core/pull/2516/commits/5148131de4c105db45327c8ee3b7f0e4cf6a3719)